package.json

```diff
@@ -138,16 +138,17 @@
     "husky": "7.0.4",
     "jest": "27.4.7",
     "lint-staged": "12.2.2",
-    "postcss": "8.4.5",
+    "postcss": "^8.4.5",
     "postcss-import": "14.0.2",
     "prettier": "2.5.1",
     "react-test-renderer": "17.0.2",
     "sqlite3": "5.0.2",
-    "tailwindcss": "2.2.19",
+    "tailwindcss": "^3.0.17",
     "ts-jest": "27.1.3",
     "ts-node": "10.4.0",
     "tsconfig-paths": "3.12.0",
-    "typescript": "4.5.5"
+    "typescript": "4.5.5",
+    "undefined": "^0.1.0"
```

Comment:
Is this`"undefined"` module meant to be here?

Reply:
I have no idea what that is or how it got there. Will remove